lib/services/web.js

```diff
@@ -32,6 +36,9 @@ var WebService = function(options) {
   this.httpsOptions = options.httpsOptions || this.node.httpsOptions;
   this.port = options.port || this.node.port || 3456;
 
+  this.enableSocketRPC = _.isUndefined(options.enableSocketRPC) ?
```

Comment:
Added in https://github.com/bitpay/bitcore-node/pull/422/commits/36f337afb31629ea035f68d906f227ecb268c61e